### PR TITLE
Feat: Updated routes references routes.rb

### PR DESCRIPTION
**Pull request by glsorensen**
## Description of Changes

Changed path references in spec files

Refactor routes page to only add routes needed so far

## Concerns (tags recommended)

Do we want to/need to reference the controllers in the routes file

## What gif describes how you feel about this PR?

the-more-you-know

### CODE REVIEW

File: app/controllers/merchant_items_controller.rb [hunk 1]
```diff
@@ -17,8 +17,9 @@ def edit
   end
 
   def show
+    # binding.pry
```
Comment (jgrazulis) [inline app/controllers/merchant_items_controller.rb]: Can you take out the pry

File: spec/features/merchant_items/show_spec.rb [hunk 2]
```diff
@@ -82,15 +83,17 @@
...
     it 'item name on index page goes to item show page' do
-      visit "/merchants/#{@merchant1.id}/items" #merchant items index
+
+      visit merchant_items_path(@merchant1)
 
       click_link "#{@item1.name}"
 
-      expect(page).to have_current_path("/merchants/#{@merchant1.id}/#{@item1.id}")
+      expect(page).to have_current_path(merchant_item_path(@merchant1, @item1))
+        # merchant_items_path(@merchant1, @item1)
```
Comment (jgrazulis) [inline spec/features/merchant_items/show_spec.rb]: Is this needed?

File: config/routes.rb [hunk 3]
```diff
@@ -1,15 +1,8 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   resources :merchants do
-    resources :items, controller: 'merchant_items'
-    resources :dashboard, controller: 'merchant_dashboard'
+    resources :items, controller: 'merchant_items', only: [:index, :show]
+    resources :dashboard, controller: 'merchant_dashboard', only: [:index]
+    resources :invoices, controller: 'merchant_invoices', only: [:index]
```
Comment (jgrazulis) [inline config/routes.rb:7]: I'm still curious about these controllers; I'm gonna make a note to ask about it at the check in.

**Review by jgrazulis — CHANGES_REQUESTED** [review]
Looks great, thanks for doing that! Once the pry is out it looks good to merge.

File: spec/features/merchant_items/show_spec.rb (hunk 2 above)
Comment (glsorensen) [reply spec/features/merchant_items/show_spec.rb]: nope

File: app/controllers/merchant_items_controller.rb (hunk 1 above)
Comment (glsorensen) [reply app/controllers/merchant_items_controller.rb]: done

File: config/routes.rb [hunk 4]
```diff
@@ -1,15 +1,8 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   resources :merchants do
-    resources :items, controller: 'merchant_items'
-    resources :dashboard, controller: 'merchant_dashboard'
+    resources :items, controller: 'merchant_items', only: [:index, :show]
```
Comment (gregoryjflaherty) [inline config/routes.rb:4]: Awesome! Are you getting any failures on your end?

[the remark this answers is not in the document]
Comment (glsorensen) [reply config/routes.rb:5]: Nope, all green

**Pull request merged**